tests/php/src/PluginSuppressionTest.php

```diff
@@ -44,6 +47,7 @@ public function setUp() {
 		add_filter(
 			'pre_http_request',
 			function( $r, $args, $url ) {
+				unset( $args );
```

Comment:
This wouldn't be relevant since `$args` is not being used or referenced anywhere in the function.

```suggestion
```

Reply:
I added it just because IntelliJ warns that the arg is unused.